torch/csrc/cuda/Graph.cpp

```diff
@@ -47,5 +47,21 @@ void THCPGraph_init(PyObject *module) {
            py::call_guard<py::gil_scoped_release>())
       .def("pool",
            &::at::cuda::CUDAGraph::pool,
-           py::call_guard<py::gil_scoped_release>());
+           py::call_guard<py::gil_scoped_release>())
+      .def(
+          "debug_dump",
+          &::at::cuda::CUDAGraph::debug_dump,
+           py::call_guard<py::gil_scoped_release>())
+
```

Comment:
is this required? `debug_dump()` takes argument `debug_path` which is handled later.

Reply:
I kept it in because that's how upstream had it
https://github.com/pytorch/pytorch/blob/main/torch/csrc/cuda/Graph.cpp#L68

Not sure if it was needed either from the reasons you gave :)  But I didn't want to deviate from upstream if I didn't have t